app/models/user.rb

```diff
@@ -2,6 +2,8 @@ class User < ApplicationRecord
 
   has_secure_password
 
+  has_many :counselors
```

Comment:
A single user would not have many counselors, though. We define the 1-1 relationship by adding `user_id` to our counselors table.